templates/vue/src/components/Combobox.vue

```diff
@@ -6,6 +6,7 @@
       :placeholder="placeholder"
       :size="size"
       :style="inputStyle"
+      autocomplete="autocomplete"
```

Comment:
Is this supposed to be:
```suggestion
      :autocomplete="autocomplete"
```
Otherwise I think it's passing the literal string "autocomplete".

Reply:
Thank you!